api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -602,6 +603,11 @@ message ResponseMapper {
   // A per mapper `body_format` to override the :ref:`body_format <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.LocalReplyConfig.body_format>`.
   // It will be used when this mapper is matched.
   config.core.v3.SubstitutionFormatString body_format_override = 4;
+
+  // HTTP headers to add to a local reply. This allows the response mapper to append, to add
+  // or to override headers of any local reply before it is sent to a downstream client.
+  repeated config.core.v3.HeaderValueOption headers_to_add = 5
+      [(validate.rules).repeated = {max_items: 1000}];
```

Comment:
I would probably remove this. The limit seems somewhat arbitrary.

Reply:
We have these limits in other places for heads to add. I think it was put in for fuzzing.

Reply:
I see, OK. Sounds good.

Reply:
Yes, this is why I added this requirement - I checked other places where the list of headers was defined and basically all of them had this limitation.